main.tf

```diff
@@ -51,6 +52,7 @@ resource "azurerm_storage_account" "lacework" {
   location                  = var.location
   resource_group_name       = azurerm_resource_group.lacework[0].name
   #enable_blob_encryption    = true
+  tags                      = azurerm_resource_group.lacework.tags
```

Comment:
Our CI tests caught an issue here:
```
Error: Missing resource instance key                                                                                                                
                                                                                                                                                    
  on ../../main.tf line 55, in resource "azurerm_storage_account" "lacework":                                                                       
  55:   tags                      = azurerm_resource_group.lacework.tags                                                                            
                                                                                                                                                    
Because azurerm_resource_group.lacework has "count" set, its attributes must                                                                        
be accessed on specific instances.                                                                                                                  
                                                                                                                                                    
For example, to correlate with indices of a referring resource, use:                                                                                
    azurerm_resource_group.lacework[count.index]                                                                                                    
```